### PR TITLE
Update overview page text

### PR DESCRIPTION
**Pull request by kevin-thankyou-lin**
Update overview page text

### CODE REVIEW

File: docs/overview.md [hunk 1]
```diff
@@ -2,32 +2,34 @@
...
 
 Data-driven algorithms, such as reinforcement learning and imitation learning, provide a powerful and generic tool in robotics. These learning paradigms, fueled by new advances in deep learning, have achieved some exciting successes in a variety of robot control problems. However, the challenges of reproducibility and the limited accessibility of robot hardware (especially during a pandemic) have impaired research progress. The overarching goal of **robosuite** is to provide researchers with:
```
Comment (snasiriany) [inline docs/overview.md:8]: remove the note about the pandemic? that's in the past now

File: docs/overview.md [hunk 2]
```diff
@@ -2,32 +2,34 @@
...
 
 ## Citation
-Please cite [robosuite](https://robosuite.ai) if you use this framework in your publications:
-```
+Please cite [**robosuite**](https://robosuite.ai) if you use this framework in your publications:
+```bibtex
 @inproceedings{robosuite2020,
   title={robosuite: A Modular Simulation Framework and Benchmark for Robot Learning},
-  author={Yuke Zhu and Josiah Wong and Ajay Mandlekar and Roberto Mart\'{i}n-Mart\'{i}n and Abhishek Joshi and Soroush Nasiriany and Yifeng Zhu},
+  author={Yuke Zhu and Josiah Wong and Ajay Mandlekar and Roberto Mart\'{i}n-Mart\'{i}n and Abhishek Joshi and Soroush Nasiriany and Yifeng Zhu and Kevin Lin},
```
Comment (snasiriany) [inline docs/overview.md:31]: is this going to be the ordering? @yukezhu @zhuyifengzju pls comment

File: robosuite/demos/demo_video_recording.py [hunk 3]
```diff
@@ -67,3 +67,4 @@
             break
 
     writer.close()
+    print(f"Saved video to {args.video_path}")
```
Comment (snasiriany) [inline robosuite/demos/demo_video_recording.py:70]: seems unrelated to this PR

**Review by snasiriany — CHANGES_REQUESTED** [review]

File: docs/overview.md (hunk 2 above)
Comment (yukezhu) [reply docs/overview.md:31]: We should follow our existing convention. New names should not be appended to the end. Instead, core members should be put in alphabetical order. See the footnote in our whitepaper: https://robosuite.ai/assets/whitepaper.pdf

**Comment by kevin-thankyou-lin** [comment]
Updated @snasiriany

**Review by snasiriany — APPROVED** [review]
LGTM
